### DISCUSSION TITLE
添加启动器用户登录

### DISCUSSION THREAD

**Discussion by shikongxingwen** [Meloong-Git/PCL#1420] [提问] [answered]
能否添加启动器用户登录（不是登录游戏，是登录启动器）
用户登陆后可以收藏mod，收藏的mod可以支持一键下载至某个游戏版本
可以记录打开游戏次数，并且在另一台电脑上登录后会进行同步

**Comment by WForst-Breeze** [comment, marked as answer]
由于维护和防御恶意攻击等诸多原因，PCL2 不会考虑支持启动器账户、社区等云服务，很抱歉
